def/taskUtils.go

```diff
@@ -1,31 +1,33 @@
 // Copyright (C) 2018 spdfg
-// 
+//
 // This file is part of Elektron.
-// 
+//
 // Elektron is free software: you can redistribute it and/or modify
 // it under the terms of the GNU General Public License as published by
 // the Free Software Foundation, either version 3 of the License, or
 // (at your option) any later version.
-// 
+//
 // Elektron is distributed in the hope that it will be useful,
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
-// 
+//
 // You should have received a copy of the GNU General Public License
 // along with Elektron.  If not, see <http://www.gnu.org/licenses/>.
-// 
+//
 
 package def
 
 import (
 	"errors"
 	"fmt"
-	"log"
 	"sort"
 
 	"github.com/mash/gokmeans"
 	"github.com/montanaflynn/stats"
+	log "github.com/sirupsen/logrus"
+	"github.com/spdfg/elektron/elektronLogging"
```

Comment:
I would alias this to _elekLog_ as you seem to be using that prefix when aliasing imports of sub-packages.

Reply:
Okay. Done